spec/patch-spec.coffee

```diff
@@ -15,7 +15,6 @@ describe "Patch", ->
         expect(iterator.next()).toEqual {value, done: false}
         expect(iterator.getInputPosition()).toEqual inputPosition, "input position for hunk #{i}"
         expect(iterator.getOutputPosition()).toEqual outputPosition, "output position for hunk #{i}"
-        return if currentSpecFailed()
```

Comment:
Jasmine 2.3 (I think) introduced the ability to run the tests random, generally. It also has the ability to fail fast. So maybe we just turn that on?